pkg/schedule/labeler/labeler.go

```diff
@@ -124,7 +124,7 @@ func (l *RegionLabeler) loadRules() error {
 		return err
 	}
 	for _, d := range toDelete {
-		if err = l.storage.DeleteRegionRule(d); err != nil {
+		if err = l.DeleteLabelRuleLocked(d); err != nil {
```

Comment:
ditto, previous, it didn't change the memory.

Reply:
we should keep storage and memory consistent

Reply:
The rule won't be add into memory.